submissions/AnthonyShostak/Hooli-popup-task/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+    <head lang="EN">
+        <title>Popup</title>
+        <link rel="stylesheet" href="css/style.css"/>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta charset="UTF-8">
+    </head>
+    <body>
+        <header class="header">
+            <nav class="nav__header">
+                <ul class="nav__list">
+                    <li class="nav__item mail"><a href="#">Mail</a></li>
+                    <li class="nav__item images"><a href="#">Images</a></li>
+                    <li class="nav__item popup__nav__item">
+                        <input type="checkbox" id="popup" class="popup_checkbox">
+                        <label for="popup">
+                            <img src="popup-icons/popup-button" alt="popup-button" class="icon popup__icon">
```

Comment:
Add extension to Image url.

Reply:
I don't understand what kind of extension I have to add. Is something works wrong?